recipes/tscv/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "tscv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db0f7d02f462013832eab786d133e2db0481e25cd6b82617cb53cecb7565ce78
+
+build:
+  number: 0
```

This looks like it's pure python, so you can do:
```suggestion
  number: 0
  noarch: python
```